packages/terra-form-select/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
   * Fixed `focus not getting removed` in IE for search and combobox variants.
   * Fixed `focus not getting removed` in IE for multiple and tag variants.
 
+* Changed
+  * Updated Jest command
```

Comment:
Do we want to combine this with the above `Changed` section or leave it like this to save a build?

Reply:
yeah, good call.